atomic_defi_design/qml/Support/Support.qml

```diff
@@ -149,12 +149,12 @@ Network fees can vary greatly depending on your selected trading pair.").arg(API
 
             FAQLine {
                 title: qsTr("Do you provide user support?")
-                text: qsTr('Yes! %1 offers support through the <a href="https://komodoplatform.com/discord">Komodo Discord server</a>. The team and the community are always happy to help!').arg(API.app_name)
+                text: qsTr('Yes! %1 offers support through the <a href="%2">%1 Discord server</a>. The team and the community are always happy to help!').arg(API.app_name).arg(API.app_discord_url)
             }
 
             FAQLine {
                 title: qsTr("Who is behind %1?").arg(API.app_name)
-                text: qsTr("%1 is developed by the Komodo team. Komodo is one of the most established blockchain projects working on innovative solutions like atomic swaps, Delayed Proof of Work, and an interoperable multi-chain architecture.").arg(API.app_name)
+                text: qsTr("%1 is developed by the Komodo team. %1 is one of the most established blockchain projects working on innovative solutions like atomic swaps, Delayed Proof of Work, and an interoperable multi-chain architecture.").arg(API.app_name)
```

Comment:
"by the Komodo team"

Reply:
and this change will not work in general, because not every %1 "is one of the most established blockchain projects working on innovative solutions like atomic swaps, Delayed Proof of Work, and an interoperable multi-chain architecture.", it's Komodo team description

Reply:
let's remove the %1 here, AtomicDEX, branded or not is developed by Komodo team.

Reply:
I suggest to just revert this FAQ entry, other ones look's good !

Reply:
Yeah I was about to ask thanks for noticing